### ISSUE TITLE
Could not find io.ktor:ktor-client-core:3.0.0-wasm2

### ISSUE THREAD

**Issue by AzraelWarrior** [jordond/connectivity#41]
Could not find io.ktor:ktor-client-core:3.0.0-wasm2

Connectivity version: 1.1.2

**Comment by jordond** [comment]
For now you need to add the experimental repository to your `settings.gradle.kts`:

```kotlin
dependencyResolutionManagement {
    repositories {
        // ... other repos

        maven("https://maven.pkg.jetbrains.space/kotlin/p/wasm/experimental")
    }
}
```

This will be fixed once the dependency is upgraded to the new ktor 3 artifact

**Comment by AzraelWarrior** [comment]
thanks a lot

**Comment by jordond** [comment]
This is now resolved with #68 and release 1.1.3 👍. You can remove the experimental repository.
